doc/tutorial.txt

```diff
@@ -66,7 +66,7 @@ $ tree -d
 
 the configure step becomes
 
-./configure --host=aarch64-linux-gnu LDFLAGS="--static -L./../json-c/." CFLAGS="-I./.." --with-deadline
+./configure --host=aarch64-linux-gnu LDFLAGS="-static -L"`pwd`"/../json-c" CFLAGS="-I./../json-c" --with-deadline
```

Comment:
there is a mismatch between CFLAGS="-Ipath to parent of json repo" and CFLAGS="-I./../json-c"

Reply:
well spotted, thanks.